subscription-manager.spec

```diff
@@ -113,16 +121,23 @@ BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
 # nesting is required since RPM requires the various preamble directives to be
 # at the start of a line making meaningful indentation impossible.
 
-Requires:  python-ethtool
-Requires:  python-iniparse
-Requires:  python-decorator
+Requires:  %{py_package_prefix}-ethtool
+Requires:  %{py_package_prefix}-iniparse
+Requires:  %{py_package_prefix}-decorator
 Requires:  virt-what
+%if %use_python3
+Requires:  python3-subscription-manager-rhsm = %{version}
```

Comment:
Why is different convection used? Why is not susbscription-manager-rhsm used too? Is it due to virt-who?

Reply:
Actually, I was trying to follow the Fedora packaging guidelines: https://fedoraproject.org/wiki/Packaging:Naming?rd=Packaging:NamingGuidelines#Python3_binary_package_naming

Reply:
OK. I didn't know about this rule.